src/Sulu/Component/Rest/Tests/Unit/DoctrineRestHelperTest.php

```diff
@@ -39,15 +39,10 @@ public function setUp(): void
 
     public function testProcessSubEntities(): void
     {
-        $entities = $this->getMockBuilder(Collection::class)->getMockForAbstractClass();
+        $entities = new ArrayCollection(['test' => true, 'hello' => null, 'foo' => false]);
 
-        $entities->expects($this->once())->method('count')->willReturn(2);
-        $entities->expects($this->once())->method('getValues')->willReturn([null, null]);
-        $entities->expects($this->once())->method('clear');
-        $entities->expects($this->once())->method('getIterator')->willReturn(new \ArrayIterator([null, null]));
-        $entities->expects($this->exactly(2))->method('add');
+        $this->restHelper->processSubEntities($entities, [], function() {});
 
-        $this->restHelper->processSubEntities($entities, [], function() {
-        });
+        $this->assertSame($entities->toArray(), [true, null, false]);
```

Comment:
I don't know what the test was supposed to test but it looks like it's just getting the values of the collection.